nimpy/py_lib.nim

```diff
@@ -212,6 +253,7 @@ proc loadPyLibFromModule(m: LibHandle): PyLib =
 
   load Py_BuildValue, "_Py_BuildValue_SizeT"
   load PyTuple_New
+  load PyTuple_Pack
```

Comment:
https://docs.python.org/3.13/c-api/tuple.html
It's also part of the stable c API, and has been present since at least 3.8